projects/cps-ui-kit/src/lib/components/cps-file-upload/cps-file-upload.component.ts

```diff
@@ -124,6 +140,11 @@ export class CpsFileUploadComponent implements OnInit {
   removeUploadedFile() {
     const name = this.uploadedFile?.name ?? '';
     this.uploadedFile = undefined;
+
+    if (this.fileInput) {
+      this.fileInput.value = '';
```

Comment:
why do we need this logic?

Reply:
no need, I had some weird behavior, tried to fix it but now it seems I can't reproduce